tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/UpgradeStep.java

```diff
@@ -47,12 +43,6 @@ public UpgradeStep(
 		_status = status;
 		_commandId = commandId;
 		_parentUpgradeStep = parentUpgradeStep;
-
-		Bundle bundle = FrameworkUtil.getBundle(UpgradeStep.class);
-
-		BundleContext bundleContext = bundle.getBundleContext();
-
-		_serviceTracker = new ServiceTracker<>(bundleContext, UpgradePlanner.class, null);
```

Comment:
Oh I forgot to open the service tracker.

_serviceTracker.open().  if you add that, then it will work.